### ISSUE TITLE
ipv6 support

### ISSUE THREAD

**Issue by engel75** [EmbarkStudios/wireguard-ui#13]
**Is your feature request related to a problem? Please describe.**
Could you please add IPv6 support?

**Comment by suom1** [comment]
This is something that we have on our to-do as we (Embark) have dual stack implemented in our infrastructure, however with this there comes some questions to take care of. 

For instance NAT vs no NAT for IPv6.

**Comment by rsheasby** [comment]
Are you open for PRs on this? This seems to be the nicest WG UI for my use case, with the two exceptions being the lack of IPv6 support, and the inability to specify custom MTUs. I'd be happy to try work on implementing those.

**Comment by suom1** [comment]
Absolutely! 

For IPv6 we would probably want to have an implementation with support for both routed IPv6 and NAT:ed IPv6. Whats your opinion on this?

**Comment by rsheasby** [comment]
Yeah I'll probably start on NAT-ed, since my cloud provider (Digital Ocean) only delegates a `/124` unfortunately. Direct routed is certainly ideal though, and absolutely the correct way to implement this.

I'll probably start on MTU config though, since that's a lot simpler and a good way to familiarise myself with the project.

**Comment by suom1** [comment]
Sounds like a good plan. The routed IPv6 should not require any more code on top of NAT:ed IPv6. Should be good enough with having `net.ipv6.conf.all.forwarding = 1` in Linux.

**Comment by fariszr** [comment]
Any updates on this ?
